<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>972</width>
    <height>599</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_7">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QLabel" name="selected_part_number_label">
        <property name="text">
         <string>Part Number:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="selected_part_number_line_edit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="selected_so_number_label">
        <property name="text">
         <string>SO Number:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="selected_so_number_line_edit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="selected_so_line_number_label">
        <property name="text">
         <string>SO Line Number:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="selected_so_line_number_line_edit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>1: Add To Cut List</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QCheckBox" name="added_to_cut_list_check_box">
           <property name="text">
            <string>Added To Cut List</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QFormLayout" name="formLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="qty_cut_label">
             <property name="text">
              <string>Qty Cut:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QSpinBox" name="qty_cut_spin_box">
             <property name="maximum">
              <number>10000</number>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="cutter_label">
             <property name="text">
              <string>Cutter:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QComboBox" name="cutter_combo_box">
             <item>
              <property name="text">
               <string>Jim</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Brandon</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Dom</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>2: Cut Data</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QCheckBox" name="is_cut_check_box">
           <property name="text">
            <string>Is Cut</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QFormLayout" name="formLayout_3">
           <item row="0" column="0">
            <widget class="QLabel" name="cut_start_label">
             <property name="text">
              <string>Cut Start:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="cut_end_label">
             <property name="text">
              <string>Cut End:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QDateEdit" name="cut_start_date_edit"/>
             </item>
             <item>
              <widget class="QLineEdit" name="cut_start_time_edit">
               <property name="inputMask">
                <string>00:00:00</string>
               </property>
               <property name="text">
                <string>00:00:00</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="1" column="1">
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QDateEdit" name="cut_end_date_edit"/>
             </item>
             <item>
              <widget class="QLineEdit" name="cut_end_time_edit">
               <property name="inputMask">
                <string>00:00:00</string>
               </property>
               <property name="text">
                <string>00:00:00</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>3: Termination Data</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QCheckBox" name="is_terminated_check_box">
           <property name="text">
            <string>Is Terminated</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QFormLayout" name="formLayout_4">
           <item row="0" column="0">
            <widget class="QLabel" name="termination_start_label">
             <property name="text">
              <string>Term Start:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="termination_end_label">
             <property name="text">
              <string>Term End:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QDateEdit" name="termination_start_date_edit"/>
             </item>
             <item>
              <widget class="QLineEdit" name="termination_start_time_edit">
               <property name="inputMask">
                <string>00:00:00</string>
               </property>
               <property name="text">
                <string>00:00:00</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="1" column="1">
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QDateEdit" name="termination_end_date_edit"/>
             </item>
             <item>
              <widget class="QLineEdit" name="termination_end_time_edit">
               <property name="inputMask">
                <string>00:00:00</string>
               </property>
               <property name="text">
                <string>00:00:00</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="title">
         <string>4: Splice Data</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QCheckBox" name="is_spliced_check_box">
           <property name="text">
            <string>Is Spliced</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QFormLayout" name="formLayout_5">
           <item row="0" column="0">
            <widget class="QLabel" name="splice_start_label">
             <property name="text">
              <string>Splice Start:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="splice_end_label">
             <property name="text">
              <string>Splice End:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QDateEdit" name="splice_start_date_edit"/>
             </item>
             <item>
              <widget class="QLineEdit" name="splice_start_time_edit">
               <property name="inputMask">
                <string>00:00:00</string>
               </property>
               <property name="text">
                <string>00:00:00</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="1" column="1">
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QDateEdit" name="splice_end_date_edit"/>
             </item>
             <item>
              <widget class="QLineEdit" name="splice_end_time_edit">
               <property name="inputMask">
                <string>00:00:00</string>
               </property>
               <property name="text">
                <string>00:00:00</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_5">
        <property name="title">
         <string>5: Other</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QCheckBox" name="is_ready_for_build_check_box">
           <property name="text">
            <string>Ready For Build</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="save_push_button">
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QFormLayout" name="formLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="so_number_label">
          <property name="text">
           <string>SO Number:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="so_number_line_edit"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="part_number_label">
          <property name="text">
           <string>Part Number:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="part_number_line_edit"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Search</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="tableWidget">
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <column>
         <property name="text">
          <string>Sales Order</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Line Number</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Customer Name</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Product Number</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Description</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Qty Left To Ship</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Qty On Cut List</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="view_push_button">
      <property name="text">
       <string>View</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>972</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionGet_Current_SO_Data_From_Fishbowl"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionGet_Current_SO_Data_From_Fishbowl">
   <property name="text">
    <string>Get Current SO Data From Fishbowl</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>added_to_cut_list_check_box</tabstop>
  <tabstop>qty_cut_spin_box</tabstop>
  <tabstop>cutter_combo_box</tabstop>
  <tabstop>is_terminated_check_box</tabstop>
  <tabstop>is_cut_check_box</tabstop>
  <tabstop>is_spliced_check_box</tabstop>
  <tabstop>is_ready_for_build_check_box</tabstop>
  <tabstop>so_number_line_edit</tabstop>
  <tabstop>part_number_line_edit</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>tableWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
